src/picasso-definition/components/point/index.js

```diff
@@ -29,5 +29,28 @@ export default function createPoint({ layoutService, colorService }) {
     beforeRender: ({ size }) => {
       windowSizeMultiplier = Math.min(size.height, size.width) / 300;
     },
+    rendererSettings: {
+      transform: () => {
+        if (chartModel.query.isPanEnded()) {
+          return false;
+        }
+        if (chartModel.query.isInteractionInProgess()) {
+          const { deltaX, deltaY } = chartModel.query.getViewState().get('dataView');
+          return {
+            horizontalScaling: 1,
+            horizontalSkewing: 0,
+            verticalSkewing: 0,
+            verticalScaling: 1,
+            horizontalMoving: deltaX,
+            verticalMoving: deltaY,
+          };
+        }
+        return false;
+      },
+      canvasBufferSize: (rect) => ({
+        width: rect.computedPhysical.width + 100,
+        height: rect.computedPhysical.height + 0,
```

Comment:
I think we should have some extra buffer both on the height and width here... otherwise points COULD be "cut off" when panning I think.

Reply:
I added 100 to the height as well.